field_records/ubna_2023.csv

```diff
@@ -116,9 +116,13 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
+2023-10-11T10:11:00 , 2023-10-18T10:38:00 , M , 007 , Central Pond , 47° 39' 16.980'' N , 122° 17' 34.470'' W , 192000 , Medium , None , None , 4.332 , 3.616 , AK , CT , MS , recover-20231018 , Ikea Batteries Nighttime 1:15-14:30 UTC; Added 2 daytime recorders (1 at Foliage and 1 at Central Pond); Added 2 new recorders M and N;
+2023-10-11T10:12:00 , 2023-10-18T10:55:00 , J , 016 , Foliage , 47° 39' 18.648'' N , 122° 17' 26.718'' W , 192000 , Medium , None , None , 4.207 , 3.599 , AK , CT , MS , recover-20231018 , Ikea Batteries Nighttime 1:15-14:30 UTC; Added 2 daytime recorders (1 at Foliage and 1 at Central Pond); Added 2 new recorders M and N;
+2023-10-11T10:13:00 , 2023-10-18T11:14:00 , I , 002 , Telephone Field , 47° 39' 25.098'' N , 122° 17' 29.220'' W , 192000 , Medium , None , None , 4.258 , 3.573 , AK , CT , MS , recover-20231018 , Panasonic Batteries Nighttime 1:15-14:30 UTC; Added 2 daytime recorders (1 at Foliage and 1 at Central Pond); Added 2 new recorders M and N;
+2023-10-11T10:14:00 , 2023-10-18T10:35:00 , N , 010 , Central Pond , 47° 39' 17.172'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.221 , 3.820 , AK , CT , MS , recover-20231018 , Ikea Batteries Daytime 0:00-24:00 UTC; Added 2 daytime recorders (1 at Foliage and 1 at Central Pond); Added 2 new recorders M and N;
+2023-10-11T10:15:00 , 2023-10-18T10:56:00 , K , 012 , Foliage , 47° 39' 18.648'' N , 122° 17' 26.718'' W , 192000 , Medium , None , None , 4.228 , 3.790 , AK , CT , MS , recover-20231018 , Panasonic Batteries Daytime 0:00-24:00 UTC; Added 2 daytime recorders (1 at Foliage and 1 at Central Pond); Added 2 new recorders M and N;
+2023-10-18T09:32:00 , (DATE-TIME) , G , 011 , E18 Bridge , 47° 39' 20.892'' N , 122° 17' 48.912'' W , 192000 , Medium , None , None , 4.144 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
+2023-10-18T09:32:00 , (DATE-TIME) , C , 009 , Foliage , 47° 39' 19.848'' N , 122° 17' 27.162'' W , 192000 , Medium , None , None , 4.137 , (VOLTAGE) , MS , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
+2023-10-18T09:33:00 , (DATE-TIME) , A , 001 , Central Pond , 47° 39' 17.022'' N , 122° 17' 34.500'' W , 192000 , Medium , None , None , 4.154 , (VOLTAGE) , MS , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
+2023-10-18T09:33:00 , (DATE-TIME) , D , 008 , Telephone Field , 47° 39' 25.182'' N , 122° 17' 29.472'' W , 192000 , Medium , None , None , 4.155 , (VOLTAGE) , MS , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
```

Comment:
> 2023-10-11T10:15:00 , 2023-10-18T10:56:00 , K , 012 , Foliage , 47° 39' 18.648'' N , 122° 17' 26.718'' W , 192000 , Medium , None , None , 4.228 , 3.790 , AK , CT , MS , recover-20231018 , Panasonic Batteries Daytime 0:00-24:00 UTC; Added 2 daytime recorders (1 at Foliage and 1 at Central Pond); Added 2 new recorders M and N;
 2023-10-18T09:32:00 , (DATE-TIME) , G , 011 , E18 Bridge , 47° 39' 20.892'' N , 122° 17' 48.912'' W , 192000 , Medium , None , None , 4.144 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
 2023-10-18T09:32:00 , (DATE-TIME) , C , 009 , Foliage , 47° 39' 19.848'' N , 122° 17' 27.162'' W , 192000 , Medium , None , None , 4.137 , (VOLTAGE) , MS , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
 2023-10-18T09:33:00 , (DATE-TIME) , A , 001 , Central Pond , 47° 39' 17.022'' N , 122° 17' 34.500'' W , 192000 , Medium , None , None , 4.154 , (VOLTAGE) , MS , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;
 2023-10-18T09:33:00 , (DATE-TIME) , D , 008 , Telephone Field , 47° 39' 25.182'' N , 122° 17' 29.472'' W , 192000 , Medium , None , None , 4.155 , (VOLTAGE) , MS , CT , MS , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; 192kHZ; Configured to 5m ON/OFF;

Here I say to remove 192kHz in the notes because it is already there in the sampling rate column